release-notes/8.0/supported-os.json

```diff
@@ -0,0 +1,345 @@
+                        "13",
+                        "12"
+                    ],
+                    "notes" : [
+                        "The iOS and tvOS simulators are supported on macOS Arm64 and x64.",
+                        "The x64 emulator (Rosetta 2) is supported on macOS Arm64.",
+                        "Mac Catalyst apps are supported on macOS Arm64 and x64."
+                    ]
+                },
+                {
+                    "id": "tvos",
+                    "name": "tvOS",
+                    "link": "https://developer.apple.com/tvos/",
+                    "lifecycle": "https://support.apple.com/apple-tv/",
+                    "architectures": [
+                        "Arm64",
+                        "x64"
+                    ],
+                    "supported-versions": [
+                        "17",
+                        "16",
+                        "15",
+                        "14",
+                        "13",
+                        "12.2"
+                    ]
+                }
+            ]
+        },
+        {
+            "name": "Linux",
+            "distributions": [
+                {
+                    "id": "alpine",
+                    "name": "Alpine",
+                    "link": "https://alpinelinux.org/",
+                    "lifecycle": "https://alpinelinux.org/releases/",
+                    "architectures": [
+                        "Arm32",
+                        "Arm64",
+                        "x64"
+                    ],
+                    "supported-versions": [
+                        "3.20",
+                        "3.19",
+                        "3.18",
+                        "3.17"
+                    ],
+                    "unsupported-versions": [
+                        "3.16"
+                    ]
+                },
+                {
+                    "id": "debian",
+                    "name": "Debian",
+                    "link": "https://www.debian.org/",
+                    "lifecycle": "https://wiki.debian.org/DebianReleases",
+                    "architectures": [
+                        "Arm32",
+                        "Arm64",
+                        "x64"
+                    ],
+                    "supported-versions": [
+                        "12"
+                    ]
+                },
+                {
+                    "id": "fedora",
+                    "name": "Fedora",
+                    "link": "https://fedoraproject.org/",
+                    "lifecycle": "https://fedoraproject.org/wiki/End_of_life",
+                    "architectures": [
+                        "Arm32",
+                        "Arm64",
+                        "x64"
+                    ],
+                    "supported-versions": [
+                        "40",
+                        "39"
+                    ],
+                    "unsupported-versions": [
+                        "38"
+                    ]
```

Comment:
Fedora is mising the `unsupported-versions` entry for `37` according to https://github.com/dotnet/core/blob/v8.0.0/release-notes/8.0/supported-os.md

Reply:
Fixed.